catroid/src/main/java/org/catrobat/catroid/ui/BaseActivity.java

```diff
@@ -51,9 +55,10 @@ protected void onCreate(Bundle savedInstanceState) {
 		titleActionBar = null;
 		returnToProjectsList = false;
 		returnByPressingBackButton = false;
-		Thread.setDefaultUncaughtExceptionHandler(new BaseExceptionHandler());
-		Utils.checkIfCrashRecoveryAndFinishActivity(this);
+		Thread.setDefaultUncaughtExceptionHandler(new BaseExceptionHandler(this));
+		checkIfCrashRecoveryAndFinishActivity(this);
 		getActionBar().setDisplayHomeAsUpEnabled(true);
+		CrashReporter.sendUnhandledCaughtException();
```

Comment:
Can't say anything about it. I do need some additional supporting comments here on its usage. Meanwhile understanding BaseActivity.java because I haven't done any work here.